core/domain/translation_fetchers_test.py

```diff
@@ -23,36 +23,54 @@
 from core.platform import models
 from core.tests import test_utils
 
+from typing import cast
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import translation_models
+
+
 (translation_models,) = models.Registry.import_models([
     models.NAMES.translation])
 
 
 class TranslationFetchersTests(test_utils.GenericTestBase):
 
-    def test_get_translation_from_model(self):
-        model_id = (
+    def test_get_translation_from_model(self) -> None:
+        # Here, create() method returns Optional[str] but for test purposes
+        # we are casting it to str. Because get() method of model only accept
+        # str type argument.
+        model_id = cast(
+            str,
```

Comment:
Instead of cast you can use `assert model_id is not None`, right?

Reply:
Cast removed, Done!